myth/Api/Server/ApiController.php

```diff
@@ -571,18 +567,24 @@ public function prevURL ($path, $clean_get = false)
 
 		if (! $clean_get)
 		{
-			if ( ! isset( $_GET ) || ! is_array( $_GET ) )
+			if (! isset($_GET) || ! is_array($_GET))
+			{
+				$_GET = [];
+			}
+
+			foreach($_GET as $key => $value)
 			{
-				$_GET = [ ];
+				$params[$key] = $value;
 			}
 
-			foreach ( $_GET as $key => $value )
```

Comment:
I think you have something off here in the PR somehow. It seems correct in the existing file that I have.